circle.yml

```diff
@@ -2,13 +2,13 @@ machine:
   pre:
     - ./scripts/circle-ci/install-docker.sh:
         pwd: $CIRCLE_PROJECT_REPONAME
-    - sudo pip install docker-compose
+    - sudo wget --retry-connrefused --waitretry=1 --read-timeout 20 --timeout 15 -t 10 -q -O /usr/local/bin/docker-compose https://github.com/docker/compose/releases/download/1.9.0/docker-compose-`uname -s`-`uname -m`
```

Comment:
Talked offline about this. We're going to leave it as 1.9.0. Keeping it at 1.9.0 also gives more reproducible builds.